Library/Homebrew/cleanup.rb

```diff
@@ -4,7 +4,7 @@
 
 module Homebrew
   module Cleanup
-    @@disk_cleanup_size = 0
+    @disk_cleanup_size = 0
```

Comment:
Just to check: does `brew cleanup` still produce the same results afterwards and is this equivalent?